objects/src/errors.rs

```diff
@@ -8,7 +9,10 @@ use core::fmt;
 pub enum AccountError {
     AccountIdInvalidFieldElement(String),
     AccountIdTooFewTrailingZeros,
+    CodeParsingFailed(ParsingError),
```

Comment:
optional nit: it might be redundant to suffix the name with `Failed`

Reply:
I think it's OK to have it like this here.